csharp/src/SeedLang/Interpreter/Opcode.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 The Aha001 Team.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace SeedLang.Interpreter {
+  // All the opcodes of Seed Virtual Machine.
```

Comment:
Seed Virtual Machine -> use "Seed VM" to make things consistent? (the dir name is "VM"). Or, a descriptive term like "the SeedLang virtual machine" could be good too.

Reply:
Change to Seed virtual machine.

BYW, I have changed the dir name to Interpreter.